src/lib/bootloader/grub2.rb

```diff
@@ -107,6 +107,7 @@ def summary(simple_mode: false)
 
       result << secure_boot_summary if Systeminfo.secure_boot_available?(name)
       result << trusted_boot_summary if Systeminfo.trusted_boot_available?(name)
+      result << update_nvram_summary if Systeminfo.nvram_available?(name)
```

Comment:
well, I start feeling that it is a bit abusing of this shortcuts. Original idea was to have in bootloader summary the most common option that user want just change by one click. I think this is quite expert option and should not be mentioned in summary page.

Reply:
that's certainly open to discussion.